docs/components/concepts/job-workers.md

```diff
@@ -94,6 +94,22 @@ A timeout may lead to two different workers working on the same job, possibly at
 
 The fact that jobs may be worked on more than once means that Zeebe is an "at least once" system with respect to job delivery and that worker code must be idempotent. In other words, workers **must** deal with jobs in a way that allows the code to be executed more than once for the same job, all while preserving the expected application state.
 
+### Timeout update
+
+When a job worker activates a job it can specify a timeout for how long the job should remain activated.
+
+However, there are jobs which have an elastic timespan, meaning they can potentially run for 5 minutes, but can be also 24+ hours and longer. That can cause a problem when the workers picking up the jobs do not know in advance how long the given process takes thus they can't accurately estimate a timeout.
+
+Also, in case of a long-running job the scenario can occur where there is a problem with the job worker but the task will be unavailable until the timeout is reached.
```

Comment:
:upside_down_face: I think this can be better formatted in a list, with each bullet point describing a different scenario. This also aligns more closely with our [technical writing guidelines](https://github.com/camunda/camunda-docs/blob/main/howtos/technical-writing-styleguide.md).

Reply:
Done, thank you for the feedback.